### PR TITLE
Preparing all the documents for our final submission.

### LINKED ISSUE #21: Create a report of our findings. `results.md`
This report must consist of (as described on the final project writeup):
- **The output and correctness of each algorithm** – You should summarize, visualize, or highlight some part of the full-scale run of each algorithm. Additionally, the report should briefly describe what tests you performed to confirm that each algorithm was working as intended.

- **The answer to your leading question** – You should direct address your proposed leading question. How did you answer this question? What did you discover? If your project was ultimately unsuccessful, give a brief reflection about what worked and what you would do differently as a team.

### PR DESCRIPTION
**Pull request by opcushing**
This includes:
- `README.md` describes the layout of the project, where important files are, etc. Also includes installation instructions. 
- `team_contract.md` is the team contract we signed at the beginning of the semester.
- `results.md` should describe our results / findings from working on this project.
- `project_proposal.md` is what `README.md` used to be: the project proposal.

### CODE REVIEW

**Review by opcushing — COMMENTED** [review]
Looks good. I think this is ready for submission

File: README.md [hunk 1]
```diff
@@ -1,57 +1,23 @@
...
-- Be able to load data in. Store it as graph
-- Create our first algorithm
-- Develop the other two algorithms
-- Develop Presentation
+# Github organization
+- The majority of our code can be found in `src/wikigraph.cc`.
+- The primary dataset can be found in `datasets/links.tsv`.
+  - Other test datasets used for test cases are included in this 'datasets' directory.
+- Tests can be found in the `tests` directory.
+- The final project presentation can be found [here](https://example.com)
```
Comment (jamieRollison) [inline README.md]: correct this link when the video is posted

**Review by jamieRollison — CHANGES_REQUESTED** [review]
good job me :) left and inline comment

**Review by jamieRollison — APPROVED** [review]
merging this myself

**Pull request merged**